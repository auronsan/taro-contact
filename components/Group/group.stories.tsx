import React from 'react';
import { Group } from './index';

export default {
  title: 'components/Group',
  component: Group,
};

export const Default = () => (
  <Group>
    <div>Child 1</div>
    <div>Child 2</div>
  </Group>
);

export const WithProps = () => (
  <Group justify="center" wrap="wrap" gap="md" p={16} mb={8} fullWidth>
    <div>Child 1</div>
    <div>Child 2</div>
  </Group>
);
