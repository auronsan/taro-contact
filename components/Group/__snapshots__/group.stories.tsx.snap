// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Group Default smoke-test 1`] = `
<main class="roboto-normal">
  <div class="group_root__JMRrN group_gap-sm__VCWmB">
    <div>
      Child 1
    </div>
    <div>
      Child 2
    </div>
  </div>
  <div class="stack_root__KVCbK toast_toast-container__8fZtQ stack_gap-xs__xTnny"
       id="toast-container"
       data-testid="toast-container"
       style="padding: 0px;"
  >
  </div>
</main>
`;

exports[`components/Group WithProps smoke-test 1`] = `
<main class="roboto-normal">
  <div class="group_root__JMRrN group_justify-center__JoiHJ group_group-wrap__6mKEO group_gap-md__ZabtE group_fullWidth__KxvAQ"
       style="padding: 16px; margin-bottom: 8px;"
  >
    <div>
      Child 1
    </div>
    <div>
      Child 2
    </div>
  </div>
  <div class="stack_root__KVCbK toast_toast-container__8fZtQ stack_gap-xs__xTnny"
       id="toast-container"
       data-testid="toast-container"
       style="padding: 0px;"
  >
  </div>
</main>
`;
